phantazm-api/src/main/java/com/github/phantazmnetwork/api/particle/data/BlockParticleData.java

```diff
@@ -24,6 +30,10 @@ public final class BlockParticleData implements ParticleData {
             Particle.FALLING_DUST.key()
     );
 
+    /**
+     * Creates a {@link ConfigProcessor} for {@link BlockParticleData}.
+     * @return A {@link ConfigProcessor} for {@link BlockParticleData}
```

Comment:
slight violation of style to have the first letter of a @return/@param capitalized, but up to you if you want to change it

Reply:
ctrl shift F says 100+ matches for both capitalized and uncapitalized...I think I'll just say "oh well" for now